### ISSUE TITLE
I would like to collaborate with your project.

### ISSUE THREAD

**Issue by tcapelle** [EvilPsyCHo/Deep-Time-Series-Prediction#2]
Have been playing with your implementation of RNN2RNN mixing it up with [this one](https://github.com/fastai/course-nlp/blob/master/7b-seq2seq-attention-translation.ipynb).
I have some remarks that have worked for me:
- Inverse the `Dense` layer dropput/linear order. `Lin(drop(x))`
- Apply the `Dense` layer to the hidden state of the encoder before sending it to the decoder.
- Have not been able to get reasonable results with the Attention layers, maybe I am doing something wrong.

**Comment by EvilPsyCHo** [comment]
hi，

    Can you give me some detail about "Have not been able to get reasonable
results with the Attention layers".

    I'll try what you said, look for the dataset validation effect.Thank
you for your feedback.


best,
zhou

Thomas Capelle <notifications@github.com> 于2020年6月19日周五 下午9:29写道：

**Comment by tcapelle** [comment]
Actually, my model get's worse.
But don't really know what params have to give to the attention layers, how many heads is reasonable.

**Comment by EvilPsyCHo** [comment]
Can you send the train & test dataset to me ?  I  will check it.

Thomas Capelle <notifications@github.com> 于2020年6月22日周一 下午4:26写道：

**Comment by tcapelle** [comment]
Not really, I am using sequences of images encoded by a CNN to generate a multi-variable timeseries.
